pwndbg-lldb.py

```diff
@@ -105,12 +105,24 @@ def find_lldb_python_path() -> str:
     if len(sys.argv) == 2:
         target = sys.argv[1]
 
+    from pwndbg.dbg.lldb.repl import PwndbgController
     from pwndbg.dbg.lldb.repl import run as run_repl
 
     if debug:
         print("[-] Launcher: Entering Pwndbg CLI")
 
-    run_repl([f"target create '{target}'"] if target else None, debug=debug)
+    def drive(startup: List[str] | None):
+        async def drive(pwndbg: PwndbgController):
```

Comment:
```suggestion
        async def drive(c: PwndbgController):
```
let's not clash with `pwndbg` import name 🤔

Reply:
Fair enough